packages/@azure/servicebus/data-plane/lib/clientEntityContext.ts

```diff
@@ -169,27 +169,19 @@ export namespace ClientEntityContext {
           );
         }
       }
-      // reconnect the batching receiver if present
+
       const batchingReceiver = entityContext.batchingReceiver;
       if (batchingReceiver && !batchingReceiver.isConnecting) {
-        try {
-          log.error(
-            "[%s] calling detached on batching receiver '%s'.",
-            connectionId,
-            batchingReceiver.name
-          );
-          await batchingReceiver.detached(error);
-        } catch (err) {
-          log.error(
-            "[%s] An error occurred while reconnecting the sender '%s': %O.",
-            connectionId,
-            batchingReceiver.name,
-            err
-          );
-        }
+        log.error(
+          "[%s] Receiver '%s' with address '%s' is a Batching Receiver, so we will not be " +
+            "re-establishing the receiver link.",
+          connectionId,
+          batchingReceiver.name,
+          batchingReceiver.address
```

Comment:
When the batching receiver is running and the network goes down, can you check if an error is thrown?

Reply:
In this case we(sdk) are not throwing an error, program exists with this error :
`Error occurred:  { Error: getaddrinfo ENOTFOUND samplesservicebus.servicebus.windows.net samplesservicebus.servicebus.windows.net:5671
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)
  name: 'ServiceCommunicationError',
  translated: true,
  retryable: false }`

Reply:
Logged #1922 to continue this exploration